mythtv/libs/libmythtv/mythplayeroverlayui.cpp

```diff
@@ -230,6 +231,7 @@ void MythPlayerOverlayUI::UpdateSliderInfo(osdInfo &Info, bool PaddedFields)
         Info.text[relPrefix + "description"] = desc;
         Info.text[relPrefix + "playedtime"] = text1;
         Info.text[relPrefix + "totaltime"] = text2;
+        Info.text[relPrefix + "wallclocktime"] = wallclock.toLocalTime().toString("ddd MMM d yyyy, hh:mm:ss");
```

Comment:
This should use the date and time formats chosen by the user.

Use gCoreContext->GetSetting("DateFormat") and gCoreContext->GetSetting("TimeFormat");

Reply:
Done.  Thanks for the clue on how to do look it up.  I assume the second arg is a default value?